test/testmachinery/system/shoot_cp_migration/migrate_test.go

```diff
@@ -181,12 +181,16 @@ func initGuestBookTest(ctx context.Context, t *ShootMigrationTest) (*application
 		ShootClient:       t.ShootClient,
 		SeedClient:        t.SourceSeedClient,
 	}
-	if !t.Shoot.Spec.Addons.NginxIngress.Enabled {
+	if t.Shoot.Spec.Addons.NginxIngress == nil || !t.Shoot.Spec.Addons.NginxIngress.Enabled {
```

Comment:
`t.Shoot.Spec.Addons` could also be `nil`

Reply:
Thanks! Fixed with 25273b1